jac-cloud/jac_cloud/tests/simple_graph.jac

```diff
@@ -803,4 +803,14 @@ walker delete_custom_object {
         # The only difference is BaseAnchor.ref doesn't
         # load the actual object and just use it as reference
     }
+}
+
+walker webhook {
```

Comment:
I'd change this to be named something else other than `webhook` to avoid confusion.